lib/rspec/mocks.rb

```diff
@@ -113,10 +113,11 @@ class << self
 
     # To speed up boot time a bit, delay loading optional or rarely
     # used features until their first use.
-    autoload :AnyInstance,      "rspec/mocks/any_instance"
-    autoload :ExpectChain,      "rspec/mocks/message_chain"
-    autoload :StubChain,        "rspec/mocks/message_chain"
-    autoload :MarshalExtension, "rspec/mocks/marshal_extension"
```

Comment:
I recall we wanted to adjust our style not to align by default, to avoid churn. I’d rather have inconsistently aligned neighbour lines, but no pressure.

Reply:
I changed and applied the rule for other lines in this file.